src/Spryker/Client/RabbitMq/RabbitMqClient.php

```diff
@@ -21,8 +23,20 @@ class RabbitMqClient extends AbstractClient implements RabbitMqClientInterface
      *
      * @return \Spryker\Client\Queue\Model\Adapter\AdapterInterface
      */
-    public function createQueueAdapter()
+    public function createQueueAdapter(): AdapterInterface
     {
         return $this->getFactory()->createQueueAdapter();
     }
+
+    /**
+     * {@inheritDoc}
+     *
+     * @api
+     *
+     * @return \Spryker\Client\RabbitMq\Model\Connection\ConnectionManagerInterface
+     */
+    public function getConnectionManager(): ConnectionManagerInterface
+    {
+        return $this->getFactory()->getStaticConnectionManager();
```

Comment:
I have my concerns with exposing connection manager in the public api. Mb we can wrap it in some other model?

Reply:
Wrapped